client/my-sites/themes/multi-site.jsx

```diff
@@ -9,12 +9,17 @@ import React from 'react';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import ThemesSiteSelectorModal from './themes-site-selector-modal';
 import { connectOptions } from './theme-options';
+import QueryUserPurchases from 'components/data/query-user-purchases';
 import ThemeShowcase from './theme-showcase';
+import userFactory from 'lib/user';
+
+const user = userFactory();
 
 const MultiSiteThemeShowcase = connectOptions(
 	( props ) => (
 		<ThemesSiteSelectorModal { ...props } sourcePath="/design">
 			<ThemeShowcase source="showcase">
+				{ <QueryUserPurchases userId={ user.get().ID } /> }
```

Comment:
My understanding is that in Multi-Site Mode, every theme shows up as purchasable because you purchase themes per-site (even if one of your sites has it purchased, you can still purchase for another site).

If that's the case, does `MultiSiteThemeShowcase` even need to fetch purchase data?